modules/hostap/src/supp_api.c

```diff
@@ -2408,6 +2408,7 @@ int supplicant_ap_wps_pbc(const struct device *dev)
 
 int supplicant_ap_wps_pin(const struct device *dev, struct wifi_wps_config_params *params)
 {
+#define WPS_PIN_EXPIRE_TIME 120
```

Comment:
do we need this to be user configurable?

Reply:
Hi @krish2718 , the macro is only used in supplicant_ap_wps_pin, so I do this so.
wps pin timeout is 120s in wpa_supplicant in wps_defs.h file, so I follow it.
/* Walk Time for push button configuration (in seconds) */
#define WPS_PBC_WALK_TIME 120

if you think using macro in function is inappropriate, I can use static,
for example:
static wps_pin_expire_time = 120.

Reply:
No, it's fine, we can revisit if we have a use case for it be modified.